packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -231,6 +262,34 @@ export function setPackagerManagerConfig(options: PackageManagerConfig, execAppO
       execFileSync('yarn', ['config', 'set', 'unsafeHttpWhitelist', '127.0.0.1'], execOptions);
       break;
     }
+    case 'yarn1': {
+      if (options.yarnVersion) {
+        execFileSync('yarn', ['set', 'version', options.yarnVersion], execOptions);
+        execFileSync('yarn', ['config', 'set', '@ama-sdk:registry', options.registry], execOptions);
+        execFileSync('yarn', ['config', 'set', '@ama-terasu:registry', options.registry], execOptions);
+        execFileSync('yarn', ['config', 'set', '@o3r:registry', options.registry], execOptions);
+        execFileSync('yarn', ['config', 'set', 'unsafeHttpWhitelist', '127.0.0.1'], execOptions);
```

Comment:
if I remember correctly, these are modifying the `~/.yarnrc` on the machine, not at project level
this probably fine for the CI but not ideal when running locally
it might be better to directly modify the file like you do line 274 instead

Reply:
true that. I'll do the updates directly on the file in the project